README.markdown

```diff
@@ -84,7 +84,7 @@ For manual install create a folder somewhere in your filesystem with the `nvm.sh
 Or if you have `git` installed, then just clone it, and check out the latest version:
 
 ```sh
-git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags origin`
+git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags --match "v[0-9]*" origin`
```

Comment:
what versions of `git` support `--match`? I don't want to raise the minimum bar for `git` support if I can avoid it.